RUNTIME ERROR: a
-------------------------------------------------
	testdata/std.flatmap5:1:21-28	function <failWith>

local failWith(x) = error x;

-------------------------------------------------
	testdata/std.flatmap5:2:10-48	builtin function <flatMap>

std.type(std.flatMap(failWith, ["a", "b", "c"]))

-------------------------------------------------
	testdata/std.flatmap5:2:1-49	builtin function <type>

std.type(std.flatMap(failWith, ["a", "b", "c"]))

-------------------------------------------------
	During evaluation	


